docs/pages/repo/docs/reference/system-variables.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: System environment variables
+description: System environment variables are built-in variables that are available to all scripts.
+---
+
+## System environment variables
+
+By setting certain environment variables, you can change Turborepo's behavior. This can be useful for creating specific configurations for different environments and machines.
+
+| Variable                           | Description                                                                                                                                                                                                                                   |
+| ---------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `TURBO_API`                        | Set the base URL for [Remote Cache](/repo/docs/core-concepts/remote-caching).                                                                                                                                                                 |
+| `TURBO_BINARY_PATH`                | Manually set the path to the `turbo` binary.                                                                                                                                                                                                  |
```

Comment:
Question: is it worth going into the gotcha's of `TURBO_BINARY_PATH` here? This was developed before global turbo so using it completely opts out of trying to find a local version.

Reply:
That sounds like a good idea